docs/src/api/class-request.md

```diff
@@ -62,13 +62,21 @@ Returns the [Frame] that initiated this request.
 **DEPRECATED** Incomplete list of headers as seen by the rendering engine. Use [`method: Request.allHeaders`] instead.
 
 ## async method: Request.headersArray
+* langs: js, python, java
 - returns: <[Array]<[Object]>>
   - `name` <[string]> Name of the header.
   - `value` <[string]> Value of the header.
 
 An array with all the request HTTP headers associated with this request. Unlike [`method: Request.allHeaders`], header names are NOT lower-cased.
 Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
 
+## async method: Request.headersArray
+* langs: csharp
+- returns: <[System.Collections.Specialized.NameValueCollection]>
+
+An array with all the request HTTP headers associated with this request. Unlike [`method: Request.allHeaders`], header names are NOT lower-cased.
```

Comment:
the comment should be automatically fetched from the inherited type. So no need to duplicate afaik.